src/Raven.Server/Utils/PoolOfThreads.cs

```diff
@@ -325,7 +325,7 @@ private void InitializeProcessThreads()
                     return;
                 }
 
-                _currentUnmanagedThreadId = PlatformDetails.GetCurrentThreadId();
+                _currentUnmanagedThreadId = NativeMemory.GetCurrentUnmanagedThreadId.Invoke();
```

Comment:
Why? This is the same.

Reply:
Server uses PAL for this. Client needs to use PlatformDetails.